etc/ci/check_no_unwrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Make sure listed files do not contain "unwrap"
+set -e
+
+cd $(git rev-parse --show-toplevel) # cd into repo root so make sure paths works in any case
+
+# files that should not contain "unwrap"
+FILES=("components/compositing/compositor.rs"
+	  "components/compositing/pipeline.rs"
```

Comment:
Sorry, looks like markdown messed up the indentation here - can you line up all of the starting quotes?